genai-perf/README.md

```diff
@@ -456,6 +456,35 @@ identify the RPC to use when sending requests to the server. (default: `None`)
 
 ### Input Options
 
+##### `--audio-length-mean <int>`
+
+The mean length of audio data in seconds. (default: `10`)
+
+##### `--audio-length-stddev <int>`
+
+The standard deviation of the length of audio data in seconds.
+(default: `0`)
+
+##### `--audio-format <str>`
+
+The format of the audio data. (default: `wav`)
```

Comment:
Post-merge comment, but should this list all valid types? As a new user reading this I would be a little confused what is valid to go here aside from the default `wav`.

Reply:
I can add supported formats on the readme.

Reply:
Perhaps both should have it? What if someone arrives at this spot of the documentation without having seen the main readme?